api/main.py

```diff
@@ -55,28 +53,7 @@ def chat():
 def chat_simplified(param=''):
     return Response(talk_to_robot_simple(PINECONE_INDEX, unquote(param)))
 
-# ---------------------- human authored content retrieval ----------------------
-
-# act as a proxy, forwarding any requests to /human/<id> to
-# https://aisafety.info/questions/<id> in order to get around CORS
-@app.route('/human/<id>', methods=['GET'])
-@cross_origin()
-def human(id):
-    import requests
-    r = requests.get(f"https://aisafety.info/questions/{id}")
-    log(f"clicked followup '{json.loads(r.text)['data']['title']}': https://stampy.ai/?state={id}")
-
-    # run a regex to replace all relative links with absolute links. Just doing
-    # a regex for now since we really don't need to parse everything out then
-    # re-serialize it for something this simple.
-    # <a href=\"/?state=6207&question=What%20is%20%22superintelligence%22%3F\">
-    #                               ⬇️
-    # <a href=\"https://stampy.ai/?state=6207&question=What%20is%20%22superintelligence%22%3F\">
-    text = re.sub(r'<a href=\\"/\?state=(\d+.*)\\">', r'<a href=\"https://aisafety.info/?state=\1\\">', r.text)
```

Comment:
Good work attempting to delete the proxy function here, but as it stands there's two issues blocking merges
1. The code to patch up link addresses is deleted without replacement. You will need either:
    - reimplement URL swapping on the front-end, or
   -  change redirection rules such that `chat.aisaftey.info/?state=X` points to the same thing as `aisafety.info/?state=X`

2. This change means we're no longer logging which followups are clicked - info that it's absolutely critical that we log. If you're intent on moving things over, you'll need to add a separate call to log stuff.

Reply:
yeah, I'll remove this stuff from this PR